File: myblog-server/src/main/java/com/site/blog/service/BlogConfigService.java
package com.site.blog.service;

import com.site.blog.model.entity.BlogConfig;
import com.baomidou.mybatisplus.extension.service.IService;

import java.util.Map;

/**
 * <p>
 *  服务类
 * </p>
 *
 */
public interface BlogConfigService extends IService<BlogConfig> {

    Map<String, String> getAllConfigs();

}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/service/BlogConfigService.kt
package com.site.blog.service

import com.baomidou.mybatisplus.extension.service.IService
import com.site.blog.model.entity.BlogConfig

/**
 *
 *
 * 服务类
 *
 *
 */
interface BlogConfigService : IService<BlogConfig?> {
    fun allConfigs(): Map<String, String>
}